Gems/Atom/Tools/ShaderManagementConsole/Code/Source/Window/ShaderManagementConsoleWindowComponent.cpp

```diff
@@ -44,12 +44,12 @@ namespace ShaderManagementConsole
 
         if (AZ::BehaviorContext* behaviorContext = azrtti_cast<AZ::BehaviorContext*>(context))
         {
-            behaviorContext->EBus<ShaderManagementConsoleWindowRequestBus>("ShaderManagementConsoleWindowRequestBus")
+            behaviorContext->EBus<ShaderManagementConsoleWindowFactoryRequestBus>("ShaderManagementConsoleWindowRequestBus")
```

Comment:
ShaderManagementConsoleWindowFactoryRequestBus should not exist and be replaced by ATFWindowFactoryRequestBus